assets/javascripts/discourse/connectors/composer-action-after/encrypt.hbs

```diff
@@ -15,10 +15,10 @@
       {{~#if model.isEncrypted}}
         {{~#unless model.editingPost~}}
           <div class="composer-action-divider"></div>
-          {{~encrypted-post-timer-dropdown
-            topicDeleteAt=this.model.topic.delete_at
-            onChange=(action "timerClicked")
-          }}
+          <EncryptedPostTimerDropdown
+            @topicDeleteAt={{this.model.topic.delete_at}}
+            {{on "change" (action "timerClicked")}}
```

Comment:
as in discourse-solved: these most likely have to be `@onChange` args (that's select-kit's api)